configs/trainer/default.yaml

```diff
@@ -10,3 +10,13 @@ max_epochs: 10
 
 # ckpt path
 resume_from_checkpoint: null
+
+# Activation learning rate finder
+# auto_lr_find=True
+
+# Precision default:32
+# precision: 32
+# Mixed precision activation https://pytorch-lightning.readthedocs.io/en/stable/advanced/mixed_precision.html?highlight=precision
+# amp_backend: apex
```

Comment:
from what I've read [here](https://discuss.pytorch.org/t/torch-distributed-data-parallel-vs-apex-distributed-data-parallel/121472), Apex is deprecated and its usage is  discouraged in lightning
![image](https://user-images.githubusercontent.com/43458894/156948034-dd20ea7e-f959-449b-89f5-95cd1e76ec75.png)